apps/docs/features/docs/Reference.generated.script.ts

```diff
@@ -0,0 +1,136 @@
+import { keyBy, isPlainObject } from 'lodash'
```

Comment:
This script now runs before dev and build, to pre-write the modified type spec and function details to generated files. Was trying to get build time down by not having to repeat this work over and over for each page.

Reply:
cool, nice!